iidm-network-impl/src/main/java/eu/itesla_project/iidm/network/impl/NodeBreakerVoltageLevel.java

```diff
@@ -904,10 +904,82 @@ public boolean disconnect(TerminalExt terminal) {
     boolean isConnected(TerminalExt terminal) {
         assert terminal instanceof NodeTerminal;
         int node = ((NodeTerminal) terminal).getNode();
-        List<TIntArrayList> paths = graph.findAllPaths(node, NodeBreakerVoltageLevel::isBusbarSection, s -> s.isOpen());
+        List<TIntArrayList> paths = graph.findAllPaths(node, NodeBreakerVoltageLevel::isBusbarSection, Switch::isOpen);
         return paths.size() > 0;
     }
 
+    void traverse(NodeTerminal terminal, VoltageLevel.TopologyTraverser traverser) {
+        traverse(terminal, traverser, new HashSet<>());
+    }
+
+    static void addNextTerminals(TerminalExt otherTerminal, List<TerminalExt> nextTerminals) {
```

Comment:
This method is used by NodeBreakerVoltageLevel and BusBreakerVoltageLevel ; should be moved in AbstractVoltageLevel class
Test if otherTerminal and nextTerminals are not null?